_datascience/deidentification.md

```diff
@@ -1,114 +1,173 @@
-    to the investigators under any circumstances, until the individuals are
-    deceased; or
-
--   There are other legal requirements prohibiting the release of the key to the investigators, until the individuals are deceased.
-
-## How can a genomic dataset be de-identified?
-
-If all direct and indirect identifiers are removed from a genomic data set, the
-genomic data itself may still be able to identify a patient depending on the
-data type and degree of processing of the specific data entity. The inclusion of
-racial, ethnic and genders in scientific research however, can be a reason to
-retain some of these indirect identifiers in the context of research datasets.
-Thus, in addition, confidentiality is supported through strong use limitations,
-data use agreements and appropriate data security.
+There are two important federal regulations related to the use of "individually
+identifiable health information." The first, the [HIPAA Prvacy
+Rule](https://www.hhs.gov/hipaa/for-professionals/privacy/index.html), restricts
+use of of [protected health
+information](https://www.hhs.gov/answers/hipaa/what-is-phi/index.html) (PHI) by
+a "covered entity." HIPAA permits a covered entity to create information that is
+not individually identifiable by following specific standards for
+de-identification. The second regulation, [Protection for Human Subjects (45 CFR
+46)](https://www.hhs.gov/ohrp/regulations-and-policy/regulations/45-cfr-46/index.html),
+also known as the Common Rule, outlines protections for individuals who
+participate in research, including protections related to data and specimens
+collected from individuals for both primary and secondary research. Research
+with data and specimens that are collected for another purpose (secondary
+research) must be de-identified to be considered not human subjects
+research.[^1]
+
+According to the [U.S. Department of Health and Human
+Services](https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html#rationale), 
+
+> The process of de-identification, by which identifiers are removed from the
+> health information, mitigates privacy risks to individuals and thereby
+> supports the secondary use of data for comparative effectiveness studies,
+> policy assessment, life sciences research, and other endeavors.
+
+More information about HIPAA compliance is available on [FHCC's IRB
+website](https://extranet.fredhutch.org/en/u/irb/hipaa-compliance.html).
+
+## Methods for creating de-identifed datasets 
+
+The HIPAA Privacy Rule provides two methods for de-identification of PHI. The
+first method, expert determination, refers to expert application and evaluation
+of methods to de-identify data. The second, safe harbor, refers to removal of
+specific identifiers from a data set.
+
+> ⚠️ While it is possible to de-identify your data yourself, unless you are an
+> expert, we suggest that you seek out an expert to help you instead. There are
```

Comment:
I wonder if it would be helpful to have an example of the nuances to explain the concern more. I have some visualizations that I used in a talk that could be helpful: https://docs.google.com/presentation/d/1kr_Lw2iHzL5fBmGga5QiQ2-z05VoqKKe1WhIjT-thFY/edit#slide=id.g147806aa961_0_22

Reply:
or just more general comments about the limitations